<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BoopathiPC" ignored="0" tests="5" failures="2" timestamp="2020-08-06T22:58:52 IST" time="0.109" errors="0">
  <testcase name="testcase1" time="0.009" classname="RetryLis">
    <skipped/>
  </testcase> <!-- testcase1 -->
  <testcase name="testcase1" time="0.003" classname="RetryLis">
    <skipped/>
  </testcase> <!-- testcase1 -->
  <testcase name="testcase1" time="0.002" classname="RetryLis">
    <skipped/>
  </testcase> <!-- testcase1 -->
  <testcase name="testcase1" time="0.001" classname="RetryLis">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at RetryLis.testcase1(RetryLis.java:9)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase name="testcase2" time="0.008" classname="RetryLis">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLis.testcase2(RetryLis.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testcase2 -->
</testsuite>
